src/Cake.Svn/SvnAliases.Info.cs

```diff
@@ -0,0 +1,293 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+using Cake.Svn.Info;
+
+namespace Cake.Svn
+{
+    public static partial class SvnAliases
+    {
+        /// <summary>
+        /// Check if the <paramref name="directoryPath"/> is a Subversion working copy.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="directoryPath">The directory.</param>
+        /// <returns>
+        /// <c>true</c> if the directory <paramref name="directoryPath"/> was added to the 
+        /// Subversion working copy, even it doesn't exist on the remote repository.
+        /// <c>false</c> if the directory exist but the directory was not added to the working copy.
+        /// </returns>
+        /// <example>
+        /// Check if the <paramref name="directoryPath"/> is a Subversion working copy.
+        /// <code>
+        /// <![CDATA[
+        ///     var isWorkingCopy = IsDirectoryInSvnWorkingCopy (@"C:\project\src\newfolder\");
+        ///
+        ///     Verbose("The directory is a working copy: {0}", isWorkingCopy);
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Info")]
+        [CakeNamespaceImport("Cake.Svn.Info")]
+        public static bool IsDirectoryInSvnWorkingCopy(this ICakeContext context, DirectoryPath directoryPath)
+        {
+            var svnInformation = new SvnInfo(context.Environment, SvnClientFactoryMethod);
+
+            return svnInformation.IsWorkingCopy(directoryPath);
+        }
+
+        /// <summary>
+        /// Check if the <paramref name="filePath"/> is a Subversion working copy.
```

Comment:
A file cannot be a working copy. Alias name indicates that it returns true for any file inside a Subversion working copy.

Comment should reflect this and behavior should also be covered by appropriate test cases.